standalonecontroller/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-standalonecontroller"
+description: "Red Hat JBoss BPM Suite 7.0 Standalone Controller container image"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite-standalonecontroller"
+    - name: "JBOSS_BPMSUITE_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+cmd:
+    - "/opt/eap/bin/standalone.sh"
+    - "-b"
+    - "0.0.0.0"
+    - "-c"
+    - "standalone-full.xml"
+scripts:
+    - package: standalonecontroller
+      exec: install
+sources:
+    - artifact: jboss-bpmsuite-7.0.0.test20170720-standalone-controller-ee7.zip
```

Comment:
This is a temporary artifact until the standalone controller product artifacts are available